src/app/profile/profile.component.html

```diff
@@ -1,3 +1,81 @@
-<p>
-  profile works!
-</p>
+
+
+<mat-tab-group [selectedIndex]="selectedIndex">
+    <mat-tab label="Updating"><div class="c-box"  >
+        <h1>Update Profile</h1>
+        <div>
+          <div>(So that we can care)</div>
+          <form [formGroup]="form" class="v-form" (submit)="onSave($event)">
+              <mat-form-field>
+                <input matInput placeholder="Weight (Kg in Number)" formControlName="weight" name="weight" type="number">
+              </mat-form-field>
+              <mat-form-field>
+                <input matInput placeholder="Height (Cm in Number)" formControlName="height" name="height" type="number">
+              </mat-form-field>
+              <mat-form-field>
+                <mat-label>Gender</mat-label> 
+                <mat-select formControlName="gender" >
+                  <mat-option value="0">Male</mat-option>
+                  <mat-option value="1">Female</mat-option>
+                </mat-select>
+              </mat-form-field>
+              <mat-form-field>
+                <input matInput placeholder="Date Of Birth" formControlName="dob" type="date">
```

Comment:
use matDatePicker

Reply:
input="date" support date formate restriction when user puts random date. It's a good choice.

will keep it as it is.